pom.xml

```diff
@@ -104,10 +102,10 @@ under the License.
       <id>cstamas</id>
       <name>Tamas Cservenak</name>
       <email>cstamas@apache.org</email>
-      <timezone>+1</timezone>
       <roles>
         <role>PMC Member</role>
       </roles>
+      <timezone>+1</timezone>
```

Comment:
This should be a timezone name, while at it. But works for me.

Reply:
Same here, I'd rather have this PR focus on just the spotless addition.  The reformat commit should not change anyy content, and I would keep both commits for reference.
So I'd rather not change anything unrelated to the formatting stuff.